consensus/enclave/impl/src/lib.rs

```diff
@@ -2340,6 +2341,330 @@ mod tests {
+                }],
+                nonce,
+                tombstone_block: 2,
+                total_mint_limit: 10000,
+            };
+            let message = prefix.hash();
+            let signature = MultiSig::new(vec![
+                governor_1.sign(message.as_ref()),
+                governor_2.sign(message.as_ref()),
+                governor_3.sign(message.as_ref()),
+            ]);
+            MintConfigTx { prefix, signature }
+        };
+
+        let recipient1 = AccountKey::random(&mut rng);
+
+        let mint_tx = create_mint_tx_to_recipient(
+            token_id1,
+            &[minter_1, minter_2, minter_3],
+            12,
+            &recipient1.default_subaddress(),
+            &mut rng,
+        );
+
+        let enclave = SgxConsensusEnclave::new(logger.clone());
+        let blockchain_config = BlockchainConfig {
+            block_version,
+            governors_map: governors_map.clone(),
+            governors_signature: sign_governors_map(&governors_map),
+            ..Default::default()
+        };
+        enclave
+            .enclave_init(
+                &Default::default(),
+                &Default::default(),
+                &None,
+                blockchain_config,
+            )
+            .unwrap();
+
+        // Initialize a ledger.
+        let sender = AccountKey::random(&mut rng);
+        let mut ledger = create_ledger();
+
+        // We want the next block that gets appended to the ledger to exceed the
+        // tombstone limit of the mint config tx, since we want to make sure
+        // that minting that relies on an old MintConfigTx (one that is past
+        // its tombstone block) still validate and mint successfully.
+        let n_blocks = mint_config_tx.prefix.tombstone_block;
+        initialize_ledger(block_version, &mut ledger, n_blocks, &sender, &mut rng);
+
+        // Form block
+        let parent_block = ledger.get_block(ledger.num_blocks().unwrap() - 1).unwrap();
+
+        let root_element = ledger.get_root_tx_out_membership_element().unwrap();
+
+        let (block, block_contents, signature) = enclave
+            .form_block(
+                &parent_block,
+                FormBlockInputs {
+                    mint_txs_with_config: vec![(
+                        mint_tx.clone(),
+                        mint_config_tx.clone(),
+                        mint_config_tx.prefix.configs[0].clone(),
+                    )],
+                    ..Default::default()
+                },
+                &root_element,
+            )
+            .unwrap();
+
+        // Verify signature
+        assert_eq!(
+            signature.signer(),
+            &enclave
+                .ake
+                .get_identity()
+                .signing_keypair
+                .lock()
+                .unwrap()
+                .public_key()
+        );
+
+        assert!(signature.verify(&block).is_ok());
+
+        // The block contents should contain two mint txs.
```

Comment:
:wrench: It looks like this is only *one* transaction.  Suggest omiting the comment
```suggestion
```